### PR TITLE
Changes after first code review

### PR DESCRIPTION
**Pull request by asharanjith**
After first code review I made following changes
*Change the name of branch.
*add meaningful name for PR.
*made PR description.

### CODE REVIEW

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @asharanjith , 

Good job so far! 👏👏👏
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- No linter errors. ✔️
- .gitignore requirements are followed. ✔️
- Professional Readme file. ✔️

 ## Required Changes ♻️


- [ ]  Pull Request should have a descriptive title telling you what changes are inside. kindly Rename your PR title to be more meaningful

- [ ] Pull Request should have a summary describing introduced changes in more detail. currently, your PR description is empty, please try to fill it with the actions that you implemented, for example : 

> 1.  Create index.html
> 2.  Setup Linters
> 3. Create .gitignore file
> 4. etc ...


## Optional suggestions
- [ ] [OPTIONAL] the branch should have a name of a feature or action (e.g. 'set-up-project' or 'create-project-files' etc ...) kindly rename the current one which is `featurebranch` .

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag  @ichala in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by asharanjith** [comment]
What is the status of the review?

File: README.md [hunk 1]
```diff
@@ -72,4 +38,4 @@ Give a ⭐️ if you like this project!
 
 This project is [MIT](./LICENSE) licensed.
```
Comment (codecaiine) [inline README.md:39]: - [ ] [OPTIONAL] Your license link is broken. Please don't forget to add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license](https://choosealicense.com/licenses/) . :smile:

**Review by codecaiine — APPROVED** [review]
Hi @asharanjith , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good project implementation :heavy_check_mark: 
- Descriptive PR :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me (@codecaiine) in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**